.jvmopts

```diff
@@ -1,4 +1,6 @@
 -Xss1m
 -Xms512m
--Xmx1200m
+-Xmx4096m
```

Comment:
also are the changes to this file necessary?

Reply:
Hmm, strange that it appears as a diff. This is what's on `master`, no?

Reply:
Rebased against the upstream and it's fine now